taskboot/docker.py

```diff
@@ -281,6 +286,88 @@ def push(self, tag):
         self.run(["push", "--state", self.state, tag])
 
 
+class DinD(Tool):
+    """
+    Interface to the Docker In Docker Taskcluster feature
+    """
+
+    def __init__(self, cache=None):
+        # Check version of remote daemon
+        self.client = really_old_docker.from_env(version=TASKCLUSTER_DIND_API_VERSION)
+        version = self.client.version()
+        assert (
```

Comment:
why do you check version here ? Can the version be different from the one passed as argument ?

Reply:
In the context of Taskcluster, the version is fixed to a really old one. If it changes, the client i'm using will fail horribly, so we want to know well before doing any damage